engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -355,6 +356,16 @@ object Main {
       .desc("Skips GraalVM and its components setup during bootstrapping.")
       .build
 
+    val executionEnvironmentOption = CliOption.builder
+      .longOpt(EXECUTION_ENVIRONMENT_OPTION)
+      .hasArg(true)
+      .numberOfArgs(1)
+      .argName("name")
+      .desc(
+        "Execution environment to use during execution (`live` or `design`)"
```

Comment:
What is the default? `design`, I think, but it shall be part of the message, I believe.